apps/decoder_json/lib/decoder/json_arrays.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Decoder.JsonArrays do
+  @moduledoc """
+  Defines a `Decoder` impl for arrays of JSON data.
```

Comment:
I think you need more here. How/why is this different than JSON lines?

Reply:
Done, how's that?

Reply:
LGTM